### ISSUE TITLE
Accessibility violation in #animation-container

### ISSUE THREAD

**Issue by kyle-cognizant** [dotlottie/player-component#125]
in `dotlottie-player.ts`:

The `#animation-container` element has `role="img"` but is lacking an `aria-label` attribute. This triggers an a11y violation: https://dequeuniversity.com/rules/axe/4.7/role-img-alt?application=axeAPI

![Screenshot 2023-06-21 at 11 55 08 AM](https://github.com/dotlottie/player-component/assets/124645198/5fd468ec-78db-4a70-9f75-c343ef763cfe)

**Comment by samuelOsborne** [comment]
Will be patched in the next released version @kyle-cognizant
